tools/deps_info.py

```diff
@@ -98,8 +98,8 @@
   # a trick to include these dependencies for setjmp even when setjmp does not
   # exist. Refer to setjmp's entry for more details.
   'emscripten_longjmp': ['malloc', 'free', 'saveSetjmp', 'setThrew'],
-  'emscripten_pc_get_file': ['emscripten_builtin_malloc', 'emscripten_builtin_free', 'emscripten_builtin_memalign', 'malloc', 'free'],
-  'emscripten_pc_get_function': ['emscripten_builtin_malloc', 'emscripten_builtin_free', 'emscripten_builtin_memalign', 'malloc', 'free'],
+  'emscripten_pc_get_file': ['malloc', 'free'],
+  'emscripten_pc_get_function': ['malloc', 'free'],
```

Comment:
I'm not sure I understand how the library.js change causes this change?

Reply:
Adding explicitly dependencies on `emscripten_builtin_malloc` etc here was never necessary because emcc.py add these dependencies explicitly:

https://github.com/emscripten-core/emscripten/blob/5c6f08fc042a07fdbe35d6d06036078011802aa4/emcc.py#L2101-L2105

However, the `test_deps_info` test was passing passing before since (before this change) these symbols existed in the final build even without any santizer flags.   Since `test_deps_info` is run without `-fsantitize` the test no longer contains these exports (yay!) which makes this test fail without this additional change.